src/mds/CDir.cc

```diff
@@ -2974,6 +2994,14 @@ int CDir::_next_dentry_on_set(set<dentry_key_t>& dns, bool missing_okay,
     // okay, we got a  dentry
     dns.erase(dnkey);
 
+    if (dn->get_projected_version() < scrub_infop->last_recursive.version &&
+	!(scrub_infop->header && scrub_infop->header->force)) {
```

Comment:
On Wed, Feb 17, 2016 at 2:54 AM, Gregory Farnum <notifications@github.com>
wrote:

> In src/mds/CDir.cc
> <https://github.com/ceph/ceph/pull/6682#discussion_r53058685>:
>
> > @@ -2974,6 +2994,14 @@ int CDir::_next_dentry_on_set(set<dentry_key_t>& dns, bool missing_okay,
> >      // okay, we got a  dentry
> >      dns.erase(dnkey);
> >
> > +    if (dn->get_projected_version() < scrub_infop->last_recursive.version &&
> > +	!(scrub_infop->header && scrub_infop->header->force)) {
>
> *bump*
>
what do you mean?



> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/6682/files#r53058685>.
>